manifests/k/KimKnight/RemoteAppTool/6.0.0.0/KimKnight.RemoteAppTool.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: KimKnight.RemoteAppTool
+PackageVersion: 6.0.0.0
+PackageLocale: en-US
+Publisher: Kim Knight

```

```suggestion
Publisher: The RemoteApp Tool developers
```
This must match Control Panel.